File: server/src/main/java/de/litexo/model/ExportModel.java
package de.litexo.model;

import de.litexo.commands.PauseCommand;
import de.litexo.commands.UnpauseCommand;
import de.litexo.events.OpenttdTerminalUpdateEvent;
import lombok.Data;

@Data
public class ExportModel {
    private OpenttdTerminalUpdateEvent openttdTerminalUpdateEvent;
    private UnpauseCommand unpauseCommand;
    private PauseCommand pauseCommand;
}

File: server/ui/src/app/api/models/export-model.ts
/* tslint:disable */
/* eslint-disable */
import { OpenttdTerminalUpdateEvent } from './openttd-terminal-update-event';
import { PauseCommand } from './pause-command';
import { ServiceError } from './service-error';
import { UnpauseCommand } from './unpause-command';
export interface ExportModel {
  openttdTerminalUpdateEvent?: OpenttdTerminalUpdateEvent;
  pauseCommand?: PauseCommand;
  serviceError?: ServiceError;
  unpauseCommand?: UnpauseCommand;
}
